### PR TITLE
Change local site infrastructure

### PR DESCRIPTION
**Pull request by ildyria**
Add ability to be a blog.

Docs are still generated from markdown, but the rest is slight more efficient.

Based on:
- https://astro.build/
- https://astrowind.vercel.app/
- https://tailwindcss.com/

still need to be done:
- support & team page
- more blog content (for later)
- refine the theme?

### CODE REVIEW

File: src/content/post/_markdown-elements-demo-post.mdx [hunk 1]
```diff
@@ -0,0 +1,204 @@
+---
+publishDate: 2023-01-02T00:00:00Z
+title: Markdown elements demo post
```
Comment (d7415) [inline src/content/post/_markdown-elements-demo-post.mdx:3]: And again

File: src/components/widgets/Announcement.astro [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  >
+  <a
+    href="https://github.com/LycheeOrg/Lychee/releases/tag/v5.5.1"
+    class="text-muted hover:underline dark:text-slate-400 font-medium">Lychee 5.5.1 is now available! »</a
+  >
+  <a
+    target="_blank"
+    rel="noopener"
+    class="ltr:ml-auto rtl:mr-auto w-[5.6rem] h-[1.25rem] ml-auto bg-contain inline-block bg-[url(https://img.shields.io/github/stars/lycheeorg/lychee.svg?style=social&label=Stars&maxAge=86400)]"
+    title="If you like AstroWind, give us a star."
```
Comment (d7415) [inline src/components/widgets/Announcement.astro:20]:
```suggestion
    title="If you like Lychee, give us a star."
```

File: src/pages/_about.astro [hunk 3]
```diff
@@ -0,0 +1,228 @@
+---
```
Comment (d7415) [inline src/pages/_about.astro:1]: This one too?

File: src/pages/_contact.astro [hunk 4]
```diff
@@ -0,0 +1,79 @@
+---
```
Comment (d7415) [inline src/pages/_contact.astro:1]: ...

File: src/pages/_homes/mobile-app.astro [hunk 5]
```diff
@@ -0,0 +1,297 @@
+---
```
Comment (d7415) [inline src/pages/_homes/mobile-app.astro:1]: ...

File: src/pages/_homes/personal.astro [hunk 6]
```diff
@@ -0,0 +1,405 @@
+---
```
Comment (d7415) [inline src/pages/_homes/personal.astro:1]: ...

File: src/pages/_homes/saas.astro [hunk 7]
```diff
@@ -0,0 +1,349 @@
+---
```
Comment (d7415) [inline src/pages/_homes/saas.astro:1]: ...

File: src/pages/_homes/startup.astro [hunk 8]
```diff
@@ -0,0 +1,317 @@
+---
```
Comment (d7415) [inline src/pages/_homes/startup.astro:1]: ...

File: src/pages/_landing/click-through.astro [hunk 9]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (d7415) [inline src/pages/_landing/click-through.astro:1]: ...

File: src/pages/_landing/lead-generation.astro [hunk 10]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (d7415) [inline src/pages/_landing/lead-generation.astro:1]: ...

File: src/pages/_landing/pre-launch.astro [hunk 11]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (d7415) [inline src/pages/_landing/pre-launch.astro:1]: ...

File: src/pages/_landing/product.astro [hunk 12]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (d7415) [inline src/pages/_landing/product.astro:1]: ...

File: src/pages/_landing/sales.astro [hunk 13]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (d7415) [inline src/pages/_landing/sales.astro:1]: ...

File: src/pages/_landing/subscription.astro [hunk 14]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (d7415) [inline src/pages/_landing/subscription.astro:1]: ...

File: src/pages/_plus.astro [hunk 15]
```diff
@@ -0,0 +1,251 @@
+---
```
Comment (d7415) [inline src/pages/_plus.astro:1]: ...

File: src/pages/_princing.astro [hunk 16]
```diff
@@ -0,0 +1,244 @@
+---
```
Comment (d7415) [inline src/pages/_princing.astro:1]: We probably don't need to talk about princing :)

File: src/pages/_privacy.md [hunk 17]
```diff
@@ -0,0 +1,185 @@
+---
```
Comment (d7415) [inline src/pages/_privacy.md:1]: ...

File: src/pages/_services.astro [hunk 18]
```diff
@@ -0,0 +1,224 @@
+---
```
Comment (d7415) [inline src/pages/_services.astro:1]: ...

File: src/pages/index.astro [hunk 19]
```diff
@@ -0,0 +1,515 @@
...
+  >
+    <Fragment slot="content">
+      <h3 class="text-2xl font-bold tracking-tight dark:text-white sm:text-3xl mb-2">Building on modern foundations</h3>
+      Gain a competitive advantage by incorporating industry leading practices
+    </Fragment>
+
+    <Fragment slot="bg">
+      <div class="absolute inset-0 bg-blue-50 dark:bg-transparent"></div>
+    </Fragment>
+  </Content> -->
```
Comment (d7415) [inline src/pages/index.astro:219]: Any need for this?
```suggestion
```

File: src/pages/index.astro [hunk 20]
```diff
@@ -0,0 +1,515 @@
...
+        description:
+          "A business big enough that it could be listed on the NASDAQ goes belly up. Disappears! It ceases to exist without me. No, you clearly don't know who you're talking to, so let me clue you in.",
+      },
+      {
+        title: 'If you could only ask one question to each person you meet, what would that question be?',
+        description:
+          "This is not about revenge. This is about justice. A lot of things can change in twelve years, Admiral. Well, that's certainly good to know. About four years. I got tired of hearing how young I looked.",
+      },
+    ]}
+  /> -->
```
Comment (d7415) [inline src/pages/index.astro:485]: There's a lot of it...
```suggestion
```

File: src/pages/rss.xml.ts [hunk 21]
```diff
@@ -0,0 +1,37 @@
...
+    return new Response(null, {
+      status: 404,
+      statusText: 'Not found',
+    });
+  }
+
+  const posts = await fetchPosts();
+
+  const rss = await getRssString({
+    title: `${SITE.name}’s Blog`,
```
Comment (d7415) [inline src/pages/rss.xml.ts:18]: Not important, but while I'm here
```suggestion
    title: `${SITE.name} Blog`,
```

File: src/pages/support.astro [hunk 22]
```diff
@@ -0,0 +1,396 @@
...
+      },
+      {
+        title: 'Call us',
+        description:
+          'Mauris faucibus finibus orci, in posuere elit viverra non. In hac habitasse platea dictumst. Cras lobortis metus a hendrerit congue.',
+        icon: 'tabler:headset',
+      },
+    ]}
+  />
+</Layout> -->
```
Comment (d7415) [inline src/pages/support.astro:395]:
```suggestion
```

File: public/decapcms/index.html [hunk 23]
```diff
@@ -0,0 +1,14 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="robots" content="noindex" />
+    <title>Content Manager</title>
+    <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+  </head>
+  <body>
+    <!-- Include the script that builds the page and powers Decap CMS -->
+    <script src="https://unpkg.com/decap-cms@^3.0.0/dist/decap-cms.js"></script>
```
Comment (d7415) [inline public/decapcms/index.html:12]: I'm assuming this is to provide an online editor for posts that will be added to the repo in markdown?

File: astro.config.mjs [hunk 24]
```diff
@@ -0,0 +1,94 @@
...
+    }),
+
+    astrowind({
+      config: './src/config.yaml',
+    }),
+  ],
+
+  image: {
+    service: squooshImageService(),
+    domains: ['cdn.pixabay.com'],
```
[the remark this answers is not in the document]
Comment (ildyria) [reply astro.config.mjs:79]: just template.

File: public/decapcms/index.html (hunk 23 above)
Comment (ildyria) [reply public/decapcms/index.html:12]: Nah, completely unrelated. good spot.

Comment (ildyria) [reply public/decapcms/index.html:12]: I didn't even realize that this was in.

File: src/config.yaml [hunk 25]
```diff
@@ -0,0 +1,73 @@
...
+    isRelatedPostsEnabled: true
+    relatedPostsCount: 4
+
+# analytics:
+#   vendors:
+#     googleAnalytics:
+#       id: null # or "G-XXXXXXXXXX"
+
+ui:
+  theme: 'light:only' # Values: "system" | "light" | "dark" | "light:only" | "dark:only"
```
[the remark this answers is not in the document]
Comment (ildyria) [reply src/config.yaml:73]: I can try the other themes, but I am no guarantees on how it looks :|

File: src/pages/index.astro (hunk 19 above)
Comment (ildyria) [reply src/pages/index.astro:219]: Not really, it is more boiler plate comment for styling later.
but can nuke :)

File: public/decapcms/index.html (hunk 23 above)
Comment (ildyria) [reply public/decapcms/index.html:12]: I'll check it out later.

**Pull request merged**